src/v/features/feature_table.h

```diff
@@ -188,6 +189,12 @@ constexpr static std::array feature_schema{
     feature::ephemeral_secrets,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{7},
+    "seeds_driven_bootstrap_capable",
```

Comment:
My intention behind this name was to emphasize that this is not the Seeds Driven Bootstap function whose availability is controlled by this `feature`, but rather some other function that depends on presence of Seeds Driven Bootstrap function in this version. If called just `seeds_driven_bootstrap`, it may be assumed that it's enabling the seeds driven bootstrap itself.